### ISSUE TITLE
Native non-HWE images

### ISSUE THREAD

**Issue by 3pings** [kairos-io/kairos#1779]
**Is your feature request related to a problem? Please describe.**
Mandating that Ubuntu images all have the HWE kernel is problematic for supporting applications that are kernel specific.

**Describe the solution you'd like**
Provide a non-HWE version of the images.

**Describe alternatives you've considered**
It is manual and tedious and seems to change with different versions of Kairos to create non-HWE images outside of the repo buildstream.  For example, now all Ubuntu versions are combined into a single Dockerfile which means retooling for those that were building custom images if we want to move forward.

**Additional context**
Non-HWE images should be provided as an option.  Not everyone wants the latest and greatest especially when using LTS in a stable environment

**Comment by mauromorales** [comment]
@3pings also, which kairos version is being used?

**Comment by mauromorales** [comment]
I'm guessing for the package `linux-image-generic` would be ok. Gautam mentioned that this would be needed only for the LTS and Saad said that they have a workaround and it is ok for this to ship on 2.4.0

**Comment by mauromorales** [comment]
Releasing and testing 2 extra Ubuntu images that only differ by the kernel will be a bit too much for our pipeline, we're already experiencing issues because of all the different versions we release. I would not want to just add them, or swap the existing ones with non-hwe kernels since it seems like a positive feature to have.

Currently we are working on the BYOI which should make the process of converting a certain image into a Kairos image a lot easier, and I guess this is the best solution to address this issue, however it will most likely not be ready for 2.4.0.

In the meantime, an alternative could be to add the two extra flavors `ubuntu-20-lts-non-hwe` and `ubuntu-22-lts-non-hwe` but without adding them to our pipelines, so they can be build manually or add to an external pipeline the way we do ours but without having to play with the original Dockerfiles. WDYT? @kairos-io/maintainers @3pings

**Comment by jimmykarily** [comment]
I agree that supporting all the different configurations in the main kairos pipelines doesn't scale. Forking kairos and tweaking the pipelines a bit to get another flavor built is easy and things will become even easier when we make some more progress with the "factory" epic lane.

**Comment by mauromorales** [comment]
A flag was added so you can now build non-HWE images as follows:

```
earthly +all --FLAVOR=ubuntu-22-lts --HWE=-non-hwe
```

This can be used for both LTS versions 20 and 22.

I'm closing this ticket, but you are welcome to re-open if there's something else needed.
